src/commands/Utility/languagestats.ts

```diff
@@ -27,7 +27,8 @@ const command: Command = {
 		const authorDb = await client.getUser(interaction.user.id)
 
 		let rawLang = interaction.options.getString("language", false)?.toLowerCase()
-		if (authorDb.lang !== "en" && authorDb.lang !== "empty" && !rawLang) rawLang = authorDb.lang
+		if (authorDb.lang !== "en" && authorDb.lang !== "empty" && !rawLang && !authorDb.lang && transformDiscordLocale(interaction.locale) !== "en")
+			rawLang = authorDb.lang ?? transformDiscordLocale(interaction.locale)
```

Comment:
Completely forgot about this but transformDiscordLocale can return "en", so you should account for that because there's no languagestats for en